tests/api.c

```diff
@@ -32755,6 +32755,59 @@ static int test_wolfSSL_check_domain(void)
 }
 
 #endif /* OPENSSL_EXTRA && HAVE_SSL_MEMIO_TESTS_DEPENDENCIES */
+#if defined(HAVE_SSL_MEMIO_TESTS_DEPENDENCIES) && \
+    !defined(WOLFSSL_SYS_CA_CERTS)
```

Comment:
I just discovered this logic is wrong and put up a PR https://github.com/wolfSSL/wolfssl/pull/8917
A simple `./configure --disable-sys-ca-certs && make` was failing.